src/components/App.tsx

```diff
@@ -2943,6 +2960,37 @@ class App extends React.Component<any, AppState> {
     }));
   });
 
+  private handleTouchStart = (event: React.TouchEvent<HTMLCanvasElement>) => {
+    event.persist(); // prevent SyntheticEvent nullification, see https://reactjs.org/docs/events.html#event-pooling
+    event.preventDefault();
+
+    touchMoving = false;
+
+    // open the context menu with the first touch's clientX and clientY
+    // if the touch is not moving
+    touchTimeout = window.setTimeout(() => {
+      if (touchMoving === false) {
+        this.openContextMenu({
+          clientX: event.touches[0].clientX,
+          clientY: event.touches[0].clientY,
+        });
+      }
+    }, 500);
```

Comment:
can you add `500` to https://github.com/excalidraw/excalidraw/blob/master/src/time_constants.ts ?